inductiva/resources/resources.py

```diff
@@ -33,11 +37,25 @@ def _machine_group_list_to_str(machine_groups) -> str:
 
 
 def list_active_machine_groups():
-    """Lists all active machine group names.
+    # pylint: disable=line-too-long
+    """Lists all active machine groups info.
+
+    This method queries Google Cloud for active machine groups
+    that belong to the user. It outputs all the information relative to
+    each machine group as folllows:
+
+    INFO:absl:Active machine groups:
+                                        Name         VM Type   # machines    Disk Size in GB       Spot         Created at
+    api-1b1f724c-5cfe-4d87-8439-9689aa139723   c2-standard-4            1                 40      False   13 Sep, 07:38:50
+    api-8e6bf7d8-4888-4de9-bda5-268484b46e6f   c2-standard-4            1                 40      False   13 Sep, 07:37:49
+
+    The name of the machine group can be used to retrieve a MachineGroup object
+    with the 'get_machine_group' function.
 
     Returns:
         List of machine group names.
     """
+    # pylint: enable=line-too-long
     try:
         api = instance_api.InstanceApi(inductiva.api.get_client())
         response = api.list_active_user_instance_groups()
```

Comment:
In this list method, I think using logging.info is a bit ugly and clutters the output, consider just printing to stdout. Again consistent with:

https://github.com/inductiva/inductiva/blob/20c315dad7cc180136d211ff5e179283569ad702/inductiva/tasks/methods.py#L133